Payara-Micro/spring-boot-jpa-jsf-example/README.md

```diff
@@ -0,0 +1,11 @@
+Compile with:
+
+    mvn compile
```

Comment:
"package" would work also, I tend to install war files to my local repository (.m2).